### DISCUSSION TITLE
Cover image not working

### DISCUSSION THREAD

**Discussion by joanamftavares** [joaomlourenco/novathesis#428] [Q&A] [answered]
Hello everyone,
I'm having the following problem:
I want to use the template for the NOVA IMS master thesis, however, it looks like this:

![image](https://user-images.githubusercontent.com/90759356/231439505-e8fcda68-9c4d-4fcc-ab1d-98a8a991d614.png)

So, instead of having the cover, it only appears the directory where it is.
Earlier this week, I was "cleaning" some of the files... I may have deleted some important file, but I can't figure out what it was...
Appreciate your help! Thanks

**Comment by joaomlourenco** [comment]
Checking what is happening…

**Reply by joanamftavares** [reply]
I'm using Overleaf so if it's easier, I could share it with you. Once again, thank you so much Professor :)

**Reply by joaomlourenco** [reply, marked as answer]
I would guess you are "compiling" in `draft`mode.  Please click in the small arrow at the right of the "compile" button and check.

If that's not the case, can you please check if the file `NOVAthesisFiles/Schools/nova/ims/Images/nova-ims-mmaa-cover.pdf` is there or is missing?  
If it is missing, just download it form https://github.com/joaomlourenco/novathesis/blob/main/NOVAthesisFiles/Schools/nova/ims/Images/nova-ims-mmaa-cover.pdf.

**Reply by joanamftavares** [reply]
I was using draft mode! Thank you so much Professor João!
